### PR TITLE
Update ko.json

### PR DESCRIPTION
**Pull request by jbsamcho**
한국어 수정

**Changes:**
korean patched

**Testing:**

### Downstream

|Affects         |Reasoning
|----------------|-------------------------------
|Documentation   |

### CODE REVIEW

**Review by undsoft — APPROVED** [review]
👍🏼

File: src/assets/i18n/ko.json [hunk 1]
```diff
@@ -1,74 +1,74 @@
...
-  "Certificate request signed": "",
-  "Certificate signing request created": "",
-  "Certificate to use for key server authentication. A valid certificate is required to verify the key server connection. WARNING: for security reasons, please protect the Certificate used for key server authentication.": "",
-  "Certificate to use when performing LDAP  certificate-based authentication. To configure LDAP certificate-based  authentication, create a Certificate Signing Request for the LDAP  provider to sign. A certificate is not required when using  username/password or Kerberos authentication.": "",
-  "Change": "",
-  "Change Enclosure Label": "",
+  "By clicking the share creation checkbox below, a new share will be created on form submission with the default share settings Additionally, local TrueNAS users will have access to the resulting share and some more configuration options will be available.": "아래의 공유 생성 체크박스를 클릭하면, 폼 제출 시 기본 공유 설정으로 새로운 공유가 생성됩니다. 또한, 로컬 TrueNAS 사용자는 생성된 공유에 액세스할 수 있으며, 몇 가지 추가 구성 옵션을 사용할 수 있습니다.",
+  "By snapshot creation time": "스냅샷 생성 시간별",
+  "CA": "CA (인증 기관)",
+  "CC": "CC": "CC (참조)",
```
Comment (undsoft) [inline src/assets/i18n/ko.json:25]: Syntax error on this line.

**Review by denysbutenko — CHANGES_REQUESTED** [review]
Please resolve merge conflict to proceed. https://github.com/truenas/webui/pull/11214 was merged recently.

**Comment by RehanY147** [comment]
Removed the syntax error and resolved conflicts. Will merge. Thanks!

**Review by denysbutenko — APPROVED** [review]

**Pull request merged**